src/thor/worker.cc

```diff
@@ -293,8 +293,8 @@ void thor_worker_t::parse_measurements(const Api& request) {
 
   // we require locations
   try {
-    auto default_accuracy = matcher->config().get<float>("gps_accuracy");
-    auto default_radius = matcher->config().get<float>("search_radius");
+    auto default_accuracy = matcher->config().emission.gps_accuracy;
```

Comment:
Let's get rid of these `auto`'s when we see them. I think in this case, it makes the code to understand.

Reply:
changed for some cases considering readability.